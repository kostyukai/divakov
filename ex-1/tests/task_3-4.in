2
1101100111111111
