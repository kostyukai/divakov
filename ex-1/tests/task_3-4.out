1001101111111111
